kafka/README.md

```diff
@@ -40,4 +40,11 @@ $ HOST_IP=$(ifconfig en0 | grep "inet " | cut -d " " -f2)
 $ docker run -d --name zookeeper --network kafka-net zookeeper:3.4
 $ docker run -d --name kafka --network kafka-net -p 9092:9092 --env ZOOKEEPER_IP=$HOST_IP --env KAFKA_ADVERTISED_HOST_NAME=$HOST_IP ches/kafka
 ```
-Connect to the Kafka broker using your host IP by updating the `bootstrap.servers` accordingly.
+Connect to the Kafka broker using your host IP.
+
+## Connecting to the server
+
+Once your containers are up and running, connect to Kafka by passing in your host and port to the Wildfly server with the environment variables `KAFFKA_HOST` and `KAFKA_PORT` respectively:
+```
+$APPSRV_HOME/bin/standalone.sh --server-config=standalone-full.xml -DKAFKA_HOST=<ip_address> -DKAFKA_PORT=<port>
```

Comment:
Can we stick to the same command which is used in the documentation of the master branch. Something like this:
$SERVER_HOME/bin/standalone.sh -c standalone-full.xml -b 0.0.0.0  -DKAFKA_HOST=<ip_address> -DKAFKA_PORT=<port>.